### PR TITLE
Completed javscript-functions exercise.

### PR DESCRIPTION
**Pull request by kevin-l1**
image

### CODE REVIEW

File: javascript-functions/main.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  return area;
+}
+const getAreaResult = getArea(17, 42);
+console.log('getArea Result:', getAreaResult);
+
+function getFirstName(person) {
+  person = {
+    firstName: 'Lelouch',
+    lastName: 'Lamperouge',
+  };
```
Comment (shawnkost) [inline javascript-functions/main.js]: You don't want to define `person` inside of this function. `person` should be getting passed in when this function is called

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change requested here Kevin!

File: javascript-functions/main.js (hunk 1 above)
Comment (kevin-l1) [reply javascript-functions/main.js]: Okay, I'll correct it thanks for the help

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**